Library/include/CSP/Systems/Assets/AssetSystem.h

```diff
@@ -98,12 +98,6 @@ class CSP_API CSP_NO_DISPOSE AssetSystem : public SystemBase
 	/// @param Callback AssetCollectionResultCallback : callback when asynchronous task finishes
 	CSP_ASYNC_RESULT void GetAssetCollectionByName(const csp::common::String& AssetCollectionName, AssetCollectionResultCallback Callback);
```

Comment:
Is there any value in keeping this one around?

Reply:
Yes. This gets a single asset collection by its name and hits a different endpoint on CHS

Reply:
Ok makes sense!